3/debian-10/rootfs/opt/bitnami/scripts/moodle/entrypoint.sh

```diff
@@ -21,7 +21,11 @@ if [[ "$1" = "/opt/bitnami/scripts/moodle/run.sh" || "$1" = "/opt/bitnami/script
     info "** Starting Moodle setup **"
     /opt/bitnami/scripts/"$(web_server_type)"/setup.sh
     /opt/bitnami/scripts/php/setup.sh
-    /opt/bitnami/scripts/mysql-client/setup.sh
+    if [[ "$MOODLE_DATABASE_TYPE" = "mariadb" || "$MOODLE_DATABASE_TYPE" = "mysqli" ]]; then
+       /opt/bitnami/scripts/mysql-client/setup.sh
+    elif [[ "$MOODLE_DATABASE_TYPE" = "pgsql" ]]; then
+       /opt/bitnami/scripts/postgresql-client/setup.sh
+    fi
```

Comment:
My understanding is that the `setup.sh` for both `mysql-client` and `postgresql-client` have a check for avoiding execution if the `MYSQL_CLIENT_*` or `POSTGRESQL_CLIENT_*` environment variables are not set. Could you check this? Leaving it like this should work fine:

```suggestion
    /opt/bitnami/scripts/mysql-client/setup.sh
    /opt/bitnami/scripts/postgresql-client/setup.sh
```

Reply:
I have thrown it over, it works accordingly without the check.